.meta/mast/qwen3_4b_mast.yaml

```diff
@@ -9,6 +9,7 @@ max_res_tokens: 512
 model: "Qwen/Qwen3-4B"
 off_by_n: 1 # Off by one by default
 launcher: mast
+compile: true # Enable torch.compile for trainer/ref_model, and CUDA graphs for vLLM
```

Comment:
This file should have been removed?

Reply:
OOf, why are there so many configs. 
Yes, i missed it in https://github.com/meta-pytorch/torchforge/pull/632/files . Please just remove it.